plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -300,13 +301,19 @@ synapticsmst_device_read_board_id (SynapticsMSTDevice *device,
 		}
 		close (fd);
 	} else {
+		/* get board ID via MCU address 0x170E instead of flash access due to HDCP2.2 running */
 		if (!synapticsmst_common_rc_get_command (connection,
-							 UPDC_READ_FROM_EEPROM,
-							 2, ADDR_CUSTOMER_ID, byte,
-							 error)) {
-			g_prefix_error (error, "failed ot read EEPROM: ");
+							UPDC_READ_FROM_MEMORY,
+							((sizeof(dwData)/sizeof(dwData[0]))*4),
+							(gint)ADDR_MEMORY_ID, (guint8*)dwData,
+							error)) {
+			g_prefix_error (error, "Memory query failed: ");
 			return FALSE;
 		}
+		byte[0] = (dwData[0]>>8)&0x000000ff;
+		byte[1] = dwData[0]&0x000000ff;		
+		g_debug ("0x170E: %lx bID %x cID %x", dwData[0], byte[0], byte[1]);
```

Comment:
Since now outputting this information in the `synapticsmst_read_board_id` function I think that the other debug line (381 in this PR) can probably be removed now.

Reply:
Fixed in latest push.